tools/render-test/main.cpp

```diff
@@ -248,139 +244,123 @@ static LRESULT CALLBACK windowProc(
-
-
-    // Do initial window-creation stuff here, rather than in the renderer-specific files
-
-    HINSTANCE instance = GetModuleHandleA(0);
-    int showCommand = SW_SHOW;
-
-    // First we register a window class.
-
-    WNDCLASSEXW windowClassDesc;
-    windowClassDesc.cbSize = sizeof(windowClassDesc);
-    windowClassDesc.style = CS_OWNDC | CS_HREDRAW | CS_VREDRAW;
-    windowClassDesc.lpfnWndProc = &windowProc;
-    windowClassDesc.cbClsExtra = 0;
-    windowClassDesc.cbWndExtra = 0;
-    windowClassDesc.hInstance = instance;
-    windowClassDesc.hIcon = 0;
-    windowClassDesc.hCursor = 0;
-    windowClassDesc.hbrBackground = 0;
-    windowClassDesc.lpszMenuName = 0;
-    windowClassDesc.lpszClassName = L"HelloWorld";
-    windowClassDesc.hIconSm = 0;
-    ATOM windowClassAtom = RegisterClassExW(&windowClassDesc);
-    if(!windowClassAtom)
-    {
-        fprintf(stderr, "error: failed to register window class\n");
-        return 1;
-    }
+	// Parse command-line options
+	SLANG_RETURN_ON_FAIL(parseOptions(&argc, argv));
+
+	// Do initial window-creation stuff here, rather than in the renderer-specific files
+
+	HINSTANCE instance = GetModuleHandleA(0);
+	int showCommand = SW_SHOW;
+
+	// First we register a window class.
+
+	WNDCLASSEXW windowClassDesc;
+	windowClassDesc.cbSize = sizeof(windowClassDesc);
+	windowClassDesc.style = CS_OWNDC | CS_HREDRAW | CS_VREDRAW;
+	windowClassDesc.lpfnWndProc = &windowProc;
+	windowClassDesc.cbClsExtra = 0;
+	windowClassDesc.cbWndExtra = 0;
+	windowClassDesc.hInstance = instance;
+	windowClassDesc.hIcon = 0;
+	windowClassDesc.hCursor = 0;
+	windowClassDesc.hbrBackground = 0;
+	windowClassDesc.lpszMenuName = 0;
+	windowClassDesc.lpszClassName = L"HelloWorld";
+	windowClassDesc.hIconSm = 0;
+	ATOM windowClassAtom = RegisterClassExW(&windowClassDesc);
+	if (!windowClassAtom)
+	{
+		fprintf(stderr, "error: failed to register window class\n");
+		return SLANG_FAIL;
+	}
 
-    // Next, we create a window using that window class.
-
-    // We will create a borderless window since our screen-capture logic in GL
-    // seems to get thrown off by having to deal with a window frame.
-    DWORD windowStyle = WS_POPUP;
-    DWORD windowExtendedStyle = 0;
-
-
-    RECT windowRect = { 0, 0, gWindowWidth, gWindowHeight };
-    AdjustWindowRectEx(&windowRect, windowStyle, /*hasMenu=*/false, windowExtendedStyle);
-
-    auto width = windowRect.right - windowRect.left;
-    auto height = windowRect.bottom - windowRect.top;
-
-    LPWSTR windowName = L"Slang Render Test";
-    HWND windowHandle = CreateWindowExW(
-        windowExtendedStyle,
-        (LPWSTR)windowClassAtom,
-        windowName,
-        windowStyle,
-        0, 0, // x, y
-        width, height,
-        NULL, // parent
-        NULL, // menu
-        instance,
-        NULL);
-    if(!windowHandle)
-    {
-        fprintf(stderr, "error: failed to create window\n");
-        return 1;
-    }
+	// Next, we create a window using that window class.
```

Comment:
Seems like these spaces got turned to tabs, leading to larger diffs than necessary.